recipes/ssh2-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ssh2-python" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "e5e77801a7a0d49799e288ecbd189b4d1c23eeadcf9c6f4ce80b31b7d397babe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+  skip: true  # [win]
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc  # [unix]
```

Comment:
Please drop this as the VMs already include compilers.

Reply:
So, I removed the gcc line and both Linux and OSX builds fail with `fatal error: libssh2.h: No such file or directory`, per this [Linux build log](https://circleci.com/gh/conda-forge/staged-recipes/19285?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

After it was added, per below diff, the build succeeds per this [build log](https://circleci.com/gh/conda-forge/staged-recipes/19289?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

```
diff --git a/recipes/ssh2-python/meta.yaml b/recipes/ssh2-python/meta.yaml
index 98ca4805..25db11fa 100644
--- a/recipes/ssh2-python/meta.yaml
+++ b/recipes/ssh2-python/meta.yaml
@@ -23,6 +23,7 @@ requirements:
     - setuptools
     - libssh2 1.8.*
     - toolchain
+    - gcc  # [unix]
   run:
     - libssh2 1.8.*
```

I notice in the build logs the one with gcc has 'gcc-4.8.5-7` in its new packages while the failing one without it does not.

I also thought it was not needed, first commits did not have it, but the build fails.

Reply:
I see. So this seems more like an issue of searching the right paths for the headers (and potentially libraries as well). This can be fixed either by setting the appropriate environment variables or using `toolchain3`, which does this for you. `toolchain3` is a bit experimental at this point, but has been pretty successful thus far. If it works, we can stick with it here.